cmd/mongocli/mongocli.go

```diff
@@ -38,6 +40,59 @@ func Execute(ctx context.Context) {
 	}
 }
 
+func updateMongoCLIConfigPath() {
+	mongoCLIConfigHome, err := config.MongoCLIConfigHome()
+	if err != nil {
+		return
+	}
+
+	mongoCLIConfigPath := fmt.Sprintf("%s/%s", mongoCLIConfigHome, "config.toml")
+	f, err := os.Open(mongoCLIConfigPath) // if config.toml is already there, exit
+	if err == nil {
+		return
+	}
+	defer f.Close()
+
+	oldMongoCLIConfigHome, err := config.OldMongoCLIConfigHome()
+	if err != nil {
+		return
+	}
+
+	oldMongoCLIConfigPath := fmt.Sprintf("%s/%s", oldMongoCLIConfigHome, "mongocli.toml")
+	in, err := os.Open(oldMongoCLIConfigPath)
+	if err != nil {
+		return
+	}
+	defer in.Close()
+
+	_, _ = fmt.Fprintf(os.Stderr, `MongoCLI uses a new config path. Copying mongocli.toml content to: %s
+`, mongoCLIConfigPath)
+
+	_, err = os.Stat(oldMongoCLIConfigHome) // check if the dir is already there
```

Comment:
you are already trying to open the old config in line 62 `in, err := os.Open(oldMongoCLIConfigPath)`, if you reach this point the dir should be already there, right?

Reply:
yes, this was the wrong one as you pointed in your other comment, thx :)